client/src/components/modals_and_popovers/AriaModal.js

```diff
@@ -0,0 +1,57 @@
+import classNames from "classnames";
+import React from "react";
+import { Modal } from "react-bootstrap";
+import FocusLock from "react-focus-lock";
+
+import "./bootstrap_modal_exstension.scss";
+
+export class AriaModal extends React.Component {
```

Comment:
Eh, woops, hadn't hit submit on one of my comments when I put that review in.

Gist of it was something like this. I don't find `AriaModal` to be a useful name. For one thing, it looks like the react-bootstrap `Modal` uses all the appropriate ARIA attributes already, so all out modals/popups using that are also "AriaModal"s. Aside from that, ARIA !== a11y and a11y what we're more interested.

Anyway, I think the descriptive name for this would be something like `FocusLockedModal` because being focused locked it the distinctive property of this component.